libqtile/backend/x11/window.py

```diff
@@ -772,19 +772,20 @@ def _reset_mask(self):
         self.window.set_attribute(eventmask=self._window_mask)
 
     def _grab_click(self):
-        # Grab button 1 to focus upon click when unfocussed
+        # Grab buttons 1 - 3  to focus upon click when unfocussed
```

Comment:
Can you change comment in `_ungrab_click` too.

Reply:
done